servicepulse/intro-pending-retries.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Pending Retries Message Management in ServicePulse
+summary: Describes how ServicePulse monitors archived failed messages, and allows unarchiving archived failed messages.
+tags:
+ - ServicePulse
+reviewed: 2016-07-28
+---
+
+
+In Versions **1.6.6** and above ServicePulse includes a screen to view and manage failed messages that have been sent for retry but are pending final status.
+
+Pending retries messages can be found by going to the pending retries screen.
+
+![Pending Retries Tab](images/pending-retries.png 'width=500')
```

Comment:
Do we need a screen shot of the whole Failed Messages screen to show the user the Pending Retries link?